{{ cookiecutter.format }}/pip_install.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+import os
+
+# Run pip install with default arguments + options read from the app requirement
+# installer args path
+
+os.execv(
+    "/app/bin/python3",
+    [
+        "/app/bin/python3",
+        "-m",
+        "pip",
+        "install",
+        "--no-cache-dir",
+        "-r",
+        "requirements.txt",
+        "--target",
+        "/app/briefcase/app_packages",
+    ]
+    + [
+        arg
+        for arg in Path(__file__)
+        .parent.joinpath("pip-options.txt")
```

Comment:
It would be a bit more readable to keep the filename expression on one line:

```suggestion
        for arg in (Path(__file__).parent / "pip-options.txt")
```